Mathlib/RingTheory/PowerSeries/Basic.lean

```diff
@@ -2680,3 +2680,143 @@ theorem algebraMap_apply'' :
 #align power_series.algebra_map_apply'' PowerSeries.algebraMap_apply''
 
 end PowerSeries
+
+
+
+
```

Comment:
Thanks. I'll use your first idea. Most of the lemmas can't move much higher up the file, because they involve coercions from R[X] to R[[X]]; this also imposes the restriction that R is commutative. However it looks as though some of them could be moved to the previous `section Trunc`, and stated in more generality. Where possible, I'll do this.